### ISSUE TITLE
Template Builder - Improvements/tasks for future work

### ISSUE THREAD

**Issue by nmadruga** [openmsupply/conforma-server#852]
A few issues/improvements notices while working with TB:
- [ ] #167 If we somehow allow the same version to be included it will fail trying to load the application because there are 2 AVAILABLE versions

TO BE CONTINUED

**Comment by CarlosNZ** [comment]
> * If we somehow allow the same version to be included it will fail trying to load the application because there are 2 AVAILABLE versions

We fixed this -- the trigger already exists, but it was being disabled during snapshot import. However, I've actually made a further fix to this which is more appropriate: #850

**Comment by CarlosNZ** [comment]
Close this now?

**Comment by nmadruga** [comment]
No... I'm trying to list all issues we have for Template Builder improvements - not that we are going to work on it now. But it's good to have a list
